db/fixtures/miq_product_features.yml

```diff
@@ -6526,7 +6530,7 @@
     - :name: Switch Language
       :description: Display Language options
       :feature_type: view
-      :identifier: sui_language 
+      :identifier: sui_language
```

Comment:
I think it is better if you don't do unrelated fixes in this pull request. If you think these should be fixed, do it in a separate pull request.

Reply:
Agreed. My IDE automatically strips trailing whitespace. I'll revert those changes.